pkg/relayer/proxy/proxy_test.go

```diff
@@ -16,35 +16,59 @@ import (
 	"github.com/pokt-network/poktroll/pkg/relayer/proxy"
 	"github.com/pokt-network/poktroll/testutil/testproxy"
 	servicetypes "github.com/pokt-network/poktroll/x/service/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+const blockHeight = 1
+
 var (
-	// ValidPayload is a helper variable for tests that need a valid payload
-	validPayload = []byte(`{"method":"someMethod","id":1,"jsonrpc":"2.0","params":["someParam"]}`)
+	// validJsonRPCPayload is a helper variable for tests that need a valid payload
+	validJsonRPCPayload []byte
 
-	supplierKeyName = "supplierKeyName"
-	appPrivateKey   = secp256k1.GenPrivKey()
-	// TODO_TECHDEBT: Source relayerProxyUrl from its config file once
+	// TODO_TECHDEBT(@okdas, @red-0ne): Source relayerProxyUrl from its config file once
 	// RelayerProxy is building its servers from the provided config file
+	relayerProxyUrl string
+
+	supplierKeyName   string
+	supplierEndpoints []*sharedtypes.SupplierEndpoint
+	appPrivateKey     *secp256k1.PrivKey
+	proxiedServices   map[string]*url.URL
+
+	defaultRelayerProxyBehavior []func(*testproxy.TestBehavior)
+)
+
+func init() {
+	validJsonRPCPayload = []byte(`{"method":"someMethod","id":1,"jsonrpc":"2.0","params":["someParam"]}`)
```

Comment:
```suggestion
	// TODO_IMPROVE: Add support for oher types of requests as well
	validJsonRPCPayload = []byte(`{"method":"someMethod","id":1,"jsonrpc":"2.0","params":["someParam"]}`)
```

Reply:
Moved to `testutil/testproxy/jsonrpc.go` under `PrepareJsonRPCRequestPayload`